andela/healthchecks-a-team
# Ft config periodic reports #151946829

### What does this PR do?
Allows User to configure either daily, weekly or monthly reports notification from the settings page.
### Description of the task to be completed.
Have Radio buttons where a user can select report notification period from.
Have a tab at the dashboard where a user can see all the reports.
### How should this be tested manually?
After starting the server `./manage.py runserver`, login into the hc-checks account and by default **Monthly** notification button is selected. Change this to the **daily reports** and save. Ensure your account is at least a _day old_. Finally run `./manage.py sendreports --loop`. You should recieve an email telling you what type of report you are recieving in this case **Daily report**
### Any background context you want to provide?
Initially, a user had only two option either to receive a monthly report about their corn jobs or not to receive them at all.
### What are the relevant pivotal tracker stories?
#151946829
